charts/rancher-istio/rancher-istio/100.0.1+up1.10.4/templates/istio-install-job.yaml

```diff
@@ -29,6 +29,10 @@ spec:
             value: {{ .Values.forceInstall | default "false" | quote }}
           - name: RELEASE_MIRROR_ENABLED
             value: {{ .Values.installer.releaseMirror.enabled | quote }}
+          - name: DEBUG
+            value: {{ .Values.installer.debug.enabled | quote }}
+          - name: SECONDS_SLEEP
```

Comment:
Should we only provide this value based on if debug is true or false?

Reply:
I’d also prefer debug to be an int that is the seconds sleep. We can address that later.